cmd/security-bootstrapper/Dockerfile

```diff
@@ -35,21 +35,22 @@ RUN make cmd/security-bootstrapper/security-bootstrapper
 FROM alpine:3.14
 
 LABEL license='SPDX-License-Identifier: Apache-2.0' \
-      copyright='Copyright (c) 2021 Intel Corporation'
+      copyright='Copyright (c) 2022 Intel Corporation'
 
 RUN apk add --update --no-cache dumb-init su-exec
 
-ENV SECURITY_INIT_DIR /edgex-init
-ARG BOOTSTRAP_REDIS_DIR=${SECURITY_INIT_DIR}/bootstrap-redis
+ENV SECURITY_INIT_STAGING_DIR /edgex-init-staging
```

Comment:
@bnevis-i , @jim-wang-intel , Isn't this a breaking change? i.e. the ENV name change now requires end user to change the name?? Also the change to `/edgex-init-staging` is breaking. 

End users/adopters will have existing compose files working with the previous release and should just be able to change the version of images to bump to the next release.

Reply:
Yep, that is probably the better approach, i.e. support both old and new way and then in 3.0 remove old way.

Reply:
SECURITY_INIT_DIR was only ever used locally, so I don't think it will break anything to change it.  But I have other problems with the PR that I will detail.  However, I think it should it should be left as it was and add the new one.